Sources/Classes/core/JavascriptBridge/MiniAppScriptMessageHandler.swift

```diff
@@ -586,6 +605,119 @@ internal class MiniAppScriptMessageHandler: NSObject, WKScriptMessageHandler {
         }
     }
 
+    func getSecureStorageItem(with callbackId: String, parameters: RequestParameters?) {
+        if let key = parameters?.secureStorageKey {
+            do {
+                if let value = try secureStorageDelegate?.get(key: key) {
+                    self.executeJavaScriptCallback(
+                        responseStatus: .onSuccess,
+                        messageId: callbackId,
+                        response: value
+                    )
+                } else {
+                    self.executeJavaScriptCallback(responseStatus: .onError, messageId: callbackId, response: prepareMAJavascriptError(MiniAppSecureStorageError.storageKeyNotFound))
```

Comment:
I think we should just return a `null` response in case the key doesn't exist.

@laubengaier @leorjoseph @munir-rakuten What do you think?

Reply:
sure makes sense, I'll update it